src/account/operations/transaction/input_selection.rs

```diff
@@ -44,32 +43,43 @@ impl AccountHandle {
         );
 
         let current_time = self.client.get_time_checked().await?;
-        let bech32_hrp = self.client.get_bech32_hrp().await?;
+
+        // Filter inputs to not include inputs that require additional outputs for storage deposit return or could be
+        // still locked TODO should maybe be done in ISA directly ?
```

Comment:
Outputs that are still locked should be skipped in the ISA

Reply:
They are now fed to forbidden outputs